packages/@ember/-internals/glimmer/tests/integration/application/rendering-test.js

```diff
@@ -38,7 +38,7 @@ moduleFor(
       });
     }
 
-    ['@test it can access the model provided by the route']() {
+    ['@feature(!EMBER_ROUTING_MODEL_ARG) it can access the model provided by the route']() {
```

Comment:
Why would this one only run when the feature is disabled? IT seems that this test should work either way to me (`this.model` is compatible with both before and after, and is not deprecated).